lib/util.js

```diff
@@ -149,3 +149,22 @@ module.exports.log = internals.consoleFn('log');
 
 
 module.exports.dir = internals.consoleFn('dir');
+
+module.exports.hash = function (values = {}) {
```

Comment:
What if we want to change it though down the line to something different? Maybe even like a custom object rather than just a blank Object?